### DISCUSSION TITLE
'cannot read range' error

### DISCUSSION THREAD

**Discussion by JM-Bader** [vdemichev/DiaNN#297] [Q&A]
I frequently get this ‘cannot read range XX-YY from file ZZ’ error in the feed, see below. 
The searches always run through and there does not appear to be an overall problem in the results
Nevertheless, 
-	What causes this error?
-	Does it have a consequence?
-	Can it be fixed?

Thanks!
![image](https://user-images.githubusercontent.com/98888664/152155874-9b2a26e2-fb6a-4bbb-a915-9dd92d31e813.png)

**Comment by vdemichev** [comment]
Happens if a dia-PASEF file is corrupted. Yes, the consequence is that some data is missing. Probably cannot be fixed. Currently DIA-NN just stops loading the file once this error occurs, so I guess almost nothing is being identified from this run. The next version will have an option to continue loading the data (at least attempt to).

Vadim
